### ISSUE TITLE
Parameter description error: pnet_tune

### ISSUE THREAD

**Issue by pbhfcycssjlmm** [richzhang/PerceptualSimilarity#131]
In your description of the pnet_tune parameter：
pnet_tune
    [False] tune the base/trunk network
    [True] keep base/trunk frozen
However：
    self.net = net_type(pretrained=not self.pnet_rand, requires_grad=self.pnet_tune)
Thus，the implementation in the code shows:
    When pnet_tune=True, set by requires_grad=self.pnet_tune, it means that the weights of the base network will be updated (that is, participate in gradient calculation).
    When pnet_tune=False, the weights of the base network will be frozen and will not participate in gradient updates.
So the correct comment should be:
    pnet_tune=True: Indicates to "unfreeze" the base network, allowing its weights to be updated during training.
    pnet_tune=False: means to "freeze" the base network and its weights will not be updated during training.
